nature.md

```diff
@@ -549,7 +545,9 @@ Some pages (e.g in checkout flows) do not have unique page titles at every stage
 <td>
 The meaning or operation of the pages on this product is generally unaffected by focus order. Additionally, in most cases, the focus order of all page content matches the visual order.
 <br><br>
-Third party scripts including advertising scripts, the Disqus commenting widget, or survey or customer services widgets (among others), may inject content where the focus order does not match the visual order. The operation and meaning of these components should be unaffected by the focus order.
+On Nature Outlook pages built with Shorthand, collapsing the main navigation into a hamburger menu may disrupt the focus order, making it harder for keyboard users to access navigation links.
```

Comment:
I don't expect most customers to know what Shorthand is, so you'll want to give a hint. 

I'd suggest saying something like "on some Nature Outlook special feature pages..." or "on some immersive special feature pages (e.g. Nature Outlook)..."

The latter might be preferable because we use Shorthand for other things too - some Milestones use it, sometimes individual journals want a fancy immersive (e.g. [this one from Nature Climate Change](https://www.nature.com/immersive/s41558-023-01832-z/index.html), Nature Magazine uses it in places, I think)

I'm going to leave the decision whether to name Shorthand directly up to you! If you do want to name them, you might also want to add a note saying that we work directly with the vendor to get stuff fixed.